javav2/example_code/athena/src/main/java/aws/example/athena/DeleteNamedQueryExample.java

```diff
@@ -47,7 +47,7 @@ public static void main(String[] args) {
             System.exit(1);
         }
 
-        String name = args[0];
+        String name ="sampleQuery" ;  //args[0];

```

Comment:
Did you intend to ignore the command line arg?

Reply:
fixed